bin/node-pg-migrate.ts

```diff
@@ -20,9 +21,25 @@ process.on('uncaughtException', (err) => {
   process.exit(1);
 });
 
+let xRequire!: NodeRequire;
+
+try {
+  xRequire = require;
+} catch (error) {
+  if (
+    error instanceof ReferenceError &&
+    error.message === 'require is not defined'
+  ) {
+    xRequire = createRequire(
+      // @ts-expect-error: ignore until esm only
+      import.meta.url
+    );
+  }
+}
+
```

Comment:
This should work fine:

```suggestion
// @ts-expect-error: ignore until esm only
const xRequire = createRequire(import.meta.url || __dirname);
```

Reply:
that's awesome
in cjs it is `{}.url || __dirname` and this works 👍 

also I love the naming `crossRequire` ❤️